android/KMAPro/kMAPro/src/main/AndroidManifest.xml

```diff
@@ -121,19 +124,7 @@
 
         <data android:scheme="file" />
         <data android:host="*" />
-
-        <!--
-                     Work around Android's ugly primitive PatternMatcher
-                     implementation that can't cope with finding a . early in
-                     the path unless it's explicitly matched.
-                -->
-        <data android:pathPattern=".*\\.kmp" />
-        <data android:pathPattern=".*\\..*\\.kmp" />
-        <data android:pathPattern=".*\\..*\\..*\\.kmp" />
-        <data android:pathPattern=".*\\..*\\..*\\..*\\.kmp" />
-        <data android:pathPattern=".*\\..*\\..*\\..*\\..*\\.kmp" />
-        <data android:pathPattern=".*\\..*\\..*\\..*\\..*\\..*\\.kmp" />
-        <data android:pathPattern=".*\\..*\\..*\\..*\\..*\\..*\\..*\\.kmp" />
+        <data android:pathPattern="/.*\\.kmp" />
```

Comment:
Can we use `android:pathSuffix=".kmp"` instead?

Reply:
From https://developer.android.com/guide/topics/manifest/data-element

pathSuffix has an API prereq

> pathSuffix and pathAdvancePattern were introduced in API level 31.

Reply:
The "/" at the start shouldn't be needed?

Reply:
Well, the slash is what the Play Store is flagging:

> Add a "/" to the beginning of the android:path attribute in the tag. The attribute might also be android:pathPrefix or android:pathPattern.

:confused:

Reply:
ugh I see